CMakeLists.txt

```diff
@@ -291,6 +292,16 @@ if(BUILD_TESTING)
     target_link_libraries(test_get_env ${PROJECT_NAME})
   endif()
 
+  rcutils_custom_add_gtest(test_env test/test_env.cpp
+    ENV
+      EMPTY_TEST=
```

Comment:
why are you adding these env vars here? They are not used in the test

Reply:
I was originally going to merge the get_env stuff in this PR - I no longer think the timing is right. Removed in a11900b.